reference/tools/tiup/playground.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Quickly Deploy a Local TiDB Cluster
+summary: Learn how to quickly deploy the local TiDB cluster using the playground component of TiUP.
+category: tools
+---
+
+# Quickly Deploy a Local TiDB Cluster
+
+The TiDB cluster is a distributed system that consists of multiple components. A typical TiDB cluster consists of at least three PD nodes, three TiKV nodes, and two TiDB nodes. If you want to try TiDB quickly, it is time-consuming and not easy to manually deploy so many components. This document introduces the playground component of TiUP and how to quickly build a local TiDB test environment using the playground component.
+
+## Overview of playground
+
+The basic usage of the playground component is shown as follows:
+
+```bash
+tiup playground [version] [flags]
+```
+
+If you directly execute the command above, the locally installed TiDB/TiKV/PD component is used or the stable version of these components start a cluster that consists of one TiKV instance, one TiDB instance, and one PD instance. The following tasks will be performed:
+
+- The version is not specified. In this situation, TiUP uses the latest version of the installed playground component. Assume that the latest version is v0.0.6, this command will work the same as `tiup playground:v0.0.6`.
+- If you have not used TiUP to install the TiDB/TiKV/PD component, the playground component uses the latest stable version of these components, and then these instances are started and operated.
+- The version of each TiDB/PD/TiKV component is not specified. By default, the latest released version of each component is used. Assume that the latest version is v4.0.0-rc, this command will work the same as `tiup playground:v0.0.6 v4.0.0-rc`.
```

Comment:
```suggestion
- Because this command does not specify the version of the TiDB, PD, and TiKV component, TiUP playground uses the latest version of each component by default. Assume that the latest version is v4.0.0-rc, then this command works the same as `tiup playground:v0.0.6 v4.0.0-rc`.
```